misc/crawl_db.js

```diff
@@ -0,0 +1,68 @@
+var Crawler = require('../src/crawler.js').Crawler
+var _ = require('lodash');
+var nconf = require('nconf');
+var Sequelize = require('sequelize');
+var Promise = Sequelize.Promise;
+var modelsFactory = require('../src/models.js');
+
+/* --------------------------------- CONSTS --------------------------------- */
+
+nconf.argv().env();
+
+/* --------------------------------- HELPERS -------------------------------- */
+
+var saveDB = exports.saveDB = function saveDB(crawlJson, dbUrl, onDone) {
```

Comment:
Consider [returning a Promise](https://github.com/petkaantonov/bluebird/blob/master/API.md#new-promisefunctionfunction-resolve-function-reject-resolver---promise) instead of taking a callback.